src/rotator_library/providers/opencode_provider.py

```
@@ -0,0 +1,154 @@
+
+        try:
+            response = await client.get(
+                f"{api_base}/models",
+                headers={"Authorization": f"Bearer {api_key}"},
+            )
+            response.raise_for_status()
+            return [
+                f"opencode/{model['id']}" for model in response.json().get("data", [])
+            ]
+        except httpx.RequestError as e:
+            lib_logger.error(f"Failed to fetch OpenCode models: {e}")
```

Ruff (0.15.4)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)